client/blocks/product-selector/README.md

```diff
@@ -51,6 +51,15 @@ The following props can be passed to the Product Selector block:
 			monthly: [ 'jetpack_backup_daily_monthly', 'jetpack_backup_realtime_monthly' ],
 		}
 		```
+	* `optionDescriptions`: ( object ) Optional descriptions of the product options. Each key is a product slug, and the value is the corresponding copy. Example:
+		```
+		optionDescriptions: {
+			jetpack_backup_daily: 'Looking for more? With Real-time backups, we save as you edit and you’ll get unlimited backup archives',
```

Comment:
For README purposes, maybe let's keep those examples short? And include a localization function?